site/content/docs/main/resource-filtering.md

```diff
@@ -88,6 +88,14 @@ Wildcard takes precedence when both a wildcard and specific resource are include
   ```bash
   velero backup create <backup-name> --selector <key>=<value>
   ```
+* Include resources that are not matching the selector
+  ```bash
+  velero backup create <backup-name> --selector <key>!=<value>
+  ```
+
+More information can be found about label selectors here :
```

Comment:
to make this align with our style guide, please update this link to use markdown and have the link embedded in the text. For example, ```For more information read the [Kuberentes label selector documentation](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors)```.

See the velero style guide for more info about link styling, https://velero.io/docs/main/style-guide/#links

Reply:
I used your example and pushed the new version. 
Thanks !